src/components/views/elements/MemberEventListSummary.js

```diff
@@ -227,6 +267,18 @@ module.exports = React.createClass({
         );
     },
 
+    _getTransitionSequence: function(events) {
+        return events.map(this._getTransition);
+    },
+
+    /**
+     * Enumerate a given membership event, `e`, where `getContent().membership` has
```

Comment:
I don't understand this description. How can you enumerate one thing?

You keep using that word. I do not think it means what you think it means.